### DISCUSSION TITLE
Slowing down attackers with resistors

### DISCUSSION THREAD

**Discussion by ochadenas** [ochadenas/cpudefense#159] [Ideas]
Hi all.

I have added resistors (in version 1.36) that slow down attackers for a certain time. They are introduced in stage 32. There are also two new heroes to increase the resistance strength and durations.

But I have found that they have only limited effect on gameplay, because even if they slow down the attackers (and thus allowing a following chip to have a 2nd or 3rd shot), they cannot change their frequency. The number of attackers per time stays the same, and this is the main problem, usually.

Well, they make the game more colourful at least.

Perhaps you can give it a try and tell what you think about it.

**Comment by ochadenas** [comment]
I've had another idea: Make the slowing effect of a resistor depend on the attacker's strength. The larger the number, the more it gets slowed down. Attackers with only 1 bit would hardly be affected, and attackers with 8 bits will become really slow. Larger resistor values increase this effect.

This could open the way for some tactical game play. It somewhat separates the stronger from the weaker attackers.

I'll try this in 1.39.

**Comment by ochadenas** [comment]
I'm still not really happy with the resistors. So in 1.40 there is another change in the mechanics:

Now resistors generate a small amount of heat. (Just as in real life.) In the game, it is not very much; unless you have lots of them, it shouldn't become annoying. The interesting thing is that there is a new hero, Claude Shannon, that converts a certain amount of the heat into information. (*) In this way, resistors can be used to gain additional cash.

Please give it a try and tell me what you think ...

(*) This has no real-world equivalent, but well, it's a game. At least Shannon _has_ worked on the concept of information and entropy, so it's not _that_ far-fetched.

**Reply by Mortis03** [reply]
How does the hero that lowers heat generation affect the resistors heat when the hero that converts resistor heat to info is in play?  Does the heat get reduced first then converted or converted then reduced?

**Reply by ochadenas** [reply]
It gets reduced first and then converted.

**Reply by Mortis03** [reply]
OK, so basically if the reduce heat hero is maxed out, the conversion of resistive heat to info has no effect?

**Reply by ochadenas** [reply]
You are right, that doesn't seem to make sense.

**Reply by ochadenas** [reply]
changed this in 1.42

**Comment by Arniiiii** [comment]
Resistors + Clause Shannon (temperature->info 20+%) + Claude Shappe (temperature reduction -100%) is overpowered combo. Like, you can have multiple resistors just to get "money-bits" and no temperature.

 Just placing multiple R 12 lvl, Clock 12 lvl, SUB 12 lvl and SHR 12 lvl is enough to complete almost any run. That's kinda interesting, but it makes ACC and MEM just useless.
